checks/evaluation/permissions/GitHubWorkflowPermissionsTopNoWrite.yml

```
@@ -0,0 +1,20 @@
+short: Checks that GitHub workflows do not have default write permissions
+desc: This rule checks that GitHub workflows do not have default write permissions
+motivation: >
+  If no permissions are declared, a workflow's GitHub token's permissions default to write for all scopes.
+  This include write permissions to push to the repository, to read encrypted secrets, etc.
+  For more information, see https://docs.github.com/en/actions/security-guides/automatic-token-authentication#permissions-for-the-github_token. 
+implementation: >
+  The rule is implemented by checking whether the `permissions` keyword is defined at the top of the workflow,
+  and that no write permissions are given.
+risk: High
+remediation:
+  effort: Low
+  text:
+    - Visit https://app.stepsecurity.io/secureworkflow/${{ repo }}/${{ workflow }}/${{ branch }}?enable=permissions
+    - Tick the 'Restrict permissions for GITHUB_TOKEN'
+    - Untick other options
+  markdown:
+    - Update your workflow using [https://app.stepsecurity.io/secureworkflow](https://app.stepsecurity.io/secureworkflow/${{ repo }}/${{ workflow }}/${{ branch }}?enable=permissions).
```

I think thats great! The following suggestion is just to keep the markdown equal to the text, besides I think "visit" works better either, considering that there are other two steps.

```suggestion
    - Visit [https://app.stepsecurity.io/secureworkflow](https://app.stepsecurity.io/secureworkflow/${{ repo }}/${{ workflow }}/${{ branch }}?enable=permissions).
```